app/components/show/controls_component.rb

```diff
@@ -83,6 +83,32 @@ def create_embargo
                               disabled: button_disabled?
     end
 
+    def create_text_extraction
+      return unless Settings.features.ocr_workflow
+      return unless ocrable? && !registered_only?
+
+      render ActionButton.new url: new_item_text_extraction_path(druid),
+                              label: 'Text extraction',
+                              open_modal: true,
+                              disabled: !open? && openable?
```

Comment:
Not totally sure about this. I think this is what we want. If it is open? we don't want to allow but I am having trouble understanding what open means.

Reply:
I think we want just `!openable?` here and I pushed a commit to change this.  Basically, the button is disabled if the object cannot be opened (which could be because it is already open and just sitting or because it happens to be  in the middle of being accessioning after it was recently closed).